src/foundry/foundry.js/applications/formApplications/userConfig.d.ts

```diff
@@ -2,15 +2,13 @@ import { ConfiguredDocumentClass } from '../../../../types/helperTypes';
 
 declare global {
   /**
-   * The User Configuration application provides a form used to allow the current client to edit preferences and
-   * configurations about a User entity (typically their own).
+   * The Application responsible for configuring a single User document.
    * @typeParam P - the type of the options object
    */
-  class UserConfig<P extends DocumentSheet.Options = DocumentSheet.Options> extends DocumentSheet<
-    P,
-    Data,
-    InstanceType<ConfiguredDocumentClass<typeof User>>
-  > {
+  class UserConfig<
+    P extends DocumentSheet.Options = Options,
+    Data extends object = UserConfig.Data<P>
+  > extends DocumentSheet<P, Data, InstanceType<ConfiguredDocumentClass<typeof User>>> {
     /** @override */
     static get defaultOptions(): Options;
```

Comment:
Please document the default similar to how this is done in other Application classes